### ISSUE TITLE
Add-AzureRmAccount : A parameter cannot be found that matches parameter name 'Credential'

### ISSUE THREAD

**Issue by jennyhunter-msft** [Azure/azure-powershell#3057]
### Cmdlet(s)
Add-AzureRmAccount

### PowerShell Version
```

Name                           Value
----                           -----
PSVersion                      6.0.0-alpha
PSEdition                      Core
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   3.0.0.0
GitCommitId                    v6.0.0-alpha.9
CLRVersion
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

### Module Version
AzureRM.NetCore.Preview

### OS Version
Ubuntu 14.0.4

### Script/Steps for Reproduction
Run Add-AzureRmAccount with the credential parameter in PowerShell for Linux after importing the AzureRM.NetCore.Preview package. Also repeatable with Login-AzureRmAccount
e.g. Add-AzureRmAccount -Credential $Credential -Tenant 'xxxx-xxxx-xxxx-xxxx' -ServicePrincipal

Expected behavior
-----------------
```
Add-AzureRmAccount -Credential $credential -TenantId $tenant -ServicePrincipal

Environment           : AzureCloud
Account               :  xxxxx-xxxxx-xxxx-xxxx
TenantId              : xxxxx-xxxxx-xxxx-xxxx
SubscriptionId        : xxxxx-xxxxx-xxxx-xxxx
SubscriptionName      : Subscription Name
CurrentStorageAccount : 

```

Actual behavior
---------------

```
Add-AzureRmAccount : A parameter cannot be found that matches parameter name 'Credential'.
At ./Azure-Demo.ps1:22 char:20
+ Add-AzureRmAccount -Credential $credential -TenantId $tenant -Service ...
+                    ~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Add-AzureRmAccount], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.Azure.Commands.Profile.AddAzureRMAccountCommand
```

**Comment by markcowl** [comment]
@jehunte  This is bad documentation, the scenarios and interface have changed due to differences in ADAL support in .net core.   The parameter set for service principal with secret login is:

```
Add-AzureRMAccount -ClientId <client-id> -Secret <secret-key> -TenantId <tenantId> [-SubscriptionId <subscriptionId>]
```

Keeping the issue to update the help documents

**Comment by markcowl** [comment]
This was fixed in this PR: #3042   WIll be released in the next version.
